initAuth.js

```diff
@@ -3,14 +3,14 @@ import { init } from "next-firebase-auth";
 const initAuth = () => {
   init({
     authPageURL: "/",
-    appPageURL: "/home",
```

Comment:
redirect to `/` page instead of `/home`

this is because the server needs a browser refresh to assign tokens to a user, so if we automatically redirect to `/home`, the user tokens will not be set until another refresh --> roles can't be checked until a user clicks on another tab 

by redirecting to `/` first, when we click on the 'proceed' button, the tokens will be assigned the first time going to `/home`, and roles can be checked as soon as the `/home` page is accessed

Reply:
our default directory is not home anymore?